pkg/error/error.go

```diff
@@ -86,72 +87,114 @@ func ConsolidateErrors(errors []ValidationError, config config.Config) []Validat
 	return append(lineLessErrors, consolidatedErrors...)
 }
 
-// FormatErrors prints the errors to the console
-func FormatErrors(errors []ValidationErrors, config config.Config) []logger.LogMessage {
+func FormatErrorsAsHumanReadable(errors []ValidationErrors, config config.Config) []logger.LogMessage {
 	var formatted_errors []logger.LogMessage
-	var codeclimateIssues []CodeclimateIssue
 
 	for _, fileErrors := range errors {
 		if len(fileErrors.Errors) > 0 {
```

Comment:
OH MY GOD the complexity of this loop

I know it's not your fault, but could you at least do this

```suggestion
		if len(fileErrors.Errors) == 0 {
			continue
		}
```

It would decrease the number of level

Reply:
done